MobiusExtras/Test/ConnectableClassTests.swift

```diff
@@ -27,33 +28,42 @@ class ConnectableTests: QuickSpec {
     // swiftlint:disable function_body_length
     override func spec() {
         describe("ConnectableClass") {
-            var errorThrown: Bool = false
-            let handleError = { (_: String) -> Void in
-                errorThrown = true
+            beforeEach {
+                MobiusHooks.setErrorHandler { message, file, line in
+                    MobiusThrowableAssertion(message: message, file: "\(file)", line: line).throw()
+                }
```

Comment:
Simplified error handler for this one test case in MobiusExtras. Perhaps I’ll look into breaking the testing error handler (and other test utilities?) out of MobiusCoreTests later.